src/compute-model-statistics/src/test/scala/VerifyComputeModelStatistics.scala

```diff
@@ -36,6 +36,33 @@ class VerifyComputeModelStatistics extends TransformerFuzzing[ComputeModelStatis
     (1, 4, 0.12, 0.34, 3)
   )).toDF(labelColumn, "col1", "col2", "col3", "col4")
 
+  test("Verify multiclass evaluation is not slow for large number of labels") {
+    val numRows = 4096
+    val numCols = 2
+    val numSlotsPerVectorCol = Array(15, 15)
+    val seed = 1337
+    val doubleCol = DatasetOptions(ColumnOptions.Scalar, DataOptions.Double)
+    val options = Map[Int, DatasetOptions](1 -> doubleCol, 2 -> doubleCol)
+    val constraints = new BasicDatasetGenerationConstraints(numRows, numCols, numSlotsPerVectorCol)
+    val labelColumn = "label"
+    val predictionColumn = SchemaConstants.SparkPredictionColumn
+    val labelsAndPrediction = GenerateDataset.generateDatasetFromOptions(session, options, constraints, seed.toLong)
+      .toDF(labelColumn, predictionColumn)
+    val scoreModelName = SchemaConstants.ScoreModelPrefix + "_test model"
+    val datasetWithLabel =
+      SparkSchema.setLabelColumnName(labelsAndPrediction, scoreModelName, labelColumn,
+        SchemaConstants.ClassificationKind)
+    val datasetWithScoredLabels =
+      SparkSchema.setScoredLabelsColumnName(datasetWithLabel, scoreModelName, predictionColumn,
+        SchemaConstants.ClassificationKind)
+
```

Comment:
Seems like alot of trouble to make a dataset, maybe juts make a scala list of tuples and parallelize with .toDF

Reply:
@imatiach-msft Could you please simplify this test, this will be tough to maintain. Maybe just make the dataset with scored labels explicitly

Reply:
done